index.html

```diff
@@ -4983,6 +4983,93 @@ <h3>Base Context</h3>
           </p>
       </section>
 
+      <section class="normative">
+        <h3>Vocabularies</h3>
+
+        <p>
+Implementations MUST ensure that the following vocabulary URLs used in the base
+context ultimately resolve to the following files, which are normative:
+        </p>
+
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>URL</th>
+              <th>Media Type</th>
+              <th>Content</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+https://www.w3.org/2018/credentials#
+              </td>
+              <td>
+text/html
+              </td>
+              <td>
+https://www.w3.org/2018/credentials/index.html
+              </td>
+            </tr>
+            <tr>
+              <td>
+https://www.w3.org/2018/credentials#
+              </td>
+              <td>
+application/ld+json
+              </td>
+              <td>
+https://www.w3.org/2018/credentials/index.jsonld
+              </td>
+            </tr>
+            <tr>
+              <td>
+https://schema.org/
+              </td>
+              <td>
+text/html
+              </td>
+              <td>
+https://schema.org/
+              </td>
+            </tr>
+            <tr>
+              <td>
+https://schema.org/
+              </td>
+              <td>
+application/ld+json
+              </td>
+              <td>
+https://schema.org/version/latest/schemaorg-current-https.jsonld
+              </td>
+            </tr>
+            <tr>
+              <td>
+https://w3id.org/security#
```

Comment:
Are we sure we want to make this one normative like this? or should we move this to a w3c origin before taking a normative dependency on it?

Reply:
This PR makes it such that it doesn't matter what URL the group decides to make normative, because it explicitly states what vocabulary is the normative one (the W3C published one). This means that the W3C VCWG is effectively taking over how the URL resolves and stating that the only correct resolution is to the W3C Vocabulary (or, whatever vocabulary the W3C VCWG deems is the one it wants to point to, like schema.org).

The benefit here is that we don't have to go around renaming all of the URLs in the ecosystem. What some in the VCWG wanted was to have control over the vocabulary URL and content, this PR does that without introducing any backwards incompatible changes.